README.adoc

```diff
@@ -211,63 +270,66 @@ You will see a login form since form authentication is implemented and configure
 
 Notice that when you sign in as Bob or Carl, the browser redirects to the `admin` page and you can view their names and roles.
 When you sign in as Alice, you can only view Alice's name.
-When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message because Dave doesn't have a role supported by the application.
+When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message 
+because Dave doesn't have a role supported by the application.
 
-Once you're done checking out the application, log out and stop the Open Liberty server:
-[role='command']
-```
-mvn liberty:stop-server
-```
 
 // =================================================================================================
 // Testing the application
 // =================================================================================================
 
 == Testing the application
 
-Write `SecurityTest` class to test the authentication and authorization of the application.
+Write [hotspot file=0]`SecurityIT` class to test the authentication and authorization of the application.
 
 [role="code_command hotspot", subs="quotes"]
 ----
-#Create the `SecurityTest` class.#
-`src/test/java/it/io/openliberty/guides/security/SecurityTest.java`
+#Create the `SecurityIT` class.#
+`src/test/java/it/io/openliberty/guides/security/SecurityIT.java`
 ----
 
-SecurityTest.java
+SecurityIT.java
 [source, java, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/test/java/it/io/openliberty/guides/security/SecurityTest.java[]
+include::finish/src/test/java/it/io/openliberty/guides/security/SecurityIT.java[]
 ----
 
-The [hotspot=testAuthenticationFail]`testAuthenticationFail()` method tests an invalid user authentication while the [hotspot=testAuthorizationFail]`testAuthorizationFail()` method tests unauthorized access to the application.
+The [hotspot=testAuthenticationFail]`testAuthenticationFail()` method tests an invalid user authentication 
+while the [hotspot=testAuthorizationFail]`testAuthorizationFail()` method tests unauthorized access to the application.
 
-The [hotspot=testAuthorizationForAdmin]`testAuthorizationForAdmin()` and [hotspot=testAuthorizationForUser]`testAuthorizationForUser()` methods, respectively, verify that a user with `admin` role and a user with `user` role are properly authenticated and can access authorized resource.
+The [hotspot=testAuthorizationForAdmin]`testAuthorizationForAdmin()` and 
+[hotspot=testAuthorizationForUser]`testAuthorizationForUser()` methods, respectively, verify that a user with `admin` role 
+and a user with `user` role are properly authenticated and can access authorized resource.
```

Comment:
Update to:
"The [hotspot=testAuthorizationForAdmin]`testAuthorizationForAdmin()` and [hotspot=testAuthorizationForUser]`testAuthorizationForUser()` methods verify that users with `admin` or `user` roles are properly authenticated and can access authorized resource."